ghost/admin/app/components/modal-tier.js

```diff
@@ -183,10 +179,17 @@ export default class ModalTierPrice extends ModalBase {
         }
 
         this.tier.set('benefits', this.benefits.filter(benefit => !benefit.get('isBlank')));
-        yield this.tier.save();
-        this.hasSaved = true;
-        yield this.confirm();
-        this.send('closeModal');
+        
+        try {
+            yield this.tier.save();
+            this.hasSaved = true;
+            yield this.confirm();
+            this.send('closeModal');
+        } catch (error) {
+            if (error === undefined) {
+                // Validation error
+            }
```

Comment:
Could you re-throw here if it's not a validation error? Just so any unexpected server errors are still caught/tracked/displayed

Reply:
Sure. Added throw error